config/routes.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+
+  get 'api/v1/laptops', to: 'laptops#index'
+  post 'api/v1/laptops', to: 'laptops#create'
+  delete 'api/v1/laptops/:id', to: 'laptops#destroy'
```

Comment:
- Rails has a tool to group all the endpoints for the same controller. If you don't need all of them, you can always add the exclusion command. I kindly invite you to improve your code using this command in routes.rb